CMakeLists.txt

```
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.22)
+file(READ VERSION VERSION)
+string(STRIP ${VERSION} VERSION_STRIPPED)
+project(
+  cps-config
+  LANGUAGES CXX
+  VERSION ${VERSION_STRIPPED})
+
+add_subdirectory(src)
+
+include(CTest)
+if(BUILD_TESTING)
+    enable_testing()
```

```suggestion
    enable_testing()
```

`include(CTest)` implies `enable_testing()`.